sdk/openai/tests.yml

```diff
@@ -8,11 +8,16 @@ extends:
       ToxTestEnv: optional
       MatrixFilters:
         - PythonVersion=3.11.*
+      MatrixReplace:
+        - TestSamples=.*/true
       UseFederatedAuth: true
       SupportedClouds: 'Public'
       EnvVars:
         AZ_OPENAI_ENDPOINT: $(AOAI-ENDPOINT)
+        AZURE_OPENAI_ENDPOINT: $(AOAI-ENDPOINT)
```

Comment:
AZURE_OPENAI_ENDPOINT env var is recognized by the openai SDK and set/deleted in our tests which test the supported env vars by the Azure client. I think this will interfere with tests - can we remove it?

Reply:
I'd like to keep the naming in samples standardized, but maybe I could acheive that with local wrappers?